pkg/realtime/metrics.go

```diff
@@ -200,197 +200,210 @@ type Metrics struct {
 	WebsocketRespHeaderBytesTotal          *prometheus.CounterVec
 }
 
+type DatacenterAggregate bool
+
+const (
+	AggregateDatacenter  DatacenterAggregate = true
+	IndividualDatacenter DatacenterAggregate = false
+)
```

Comment:
And it you take my naming suggestion then these might be named:
```
WithoutDatacenter(Label|Dimension)? 
WithDatacenter(Label|Dimension)?
```

Reply:
> Besides renaming these constants, they should probably live outside the realtime package so that they can be used to the origin/domain inspector metrics packages as well.

Could you recommend a package to add these to? I've experimented with using the `prom` package but am not sure if that is the right place.